common/statsd/statsd.h

```diff
@@ -2,6 +2,7 @@
 #define SORBET_CORE_STATSD_H
 
 #include "core/core.h"
+#include <map>
```

Comment:
I believe that `core.h` includes `common.h` which includes UnorderedMap; you could use an UnorderedMap here with no new dependencies if you wanted.

Reply:
I intentionally wanted an ordered map here, for my own sanity